packages/suite/src/support/messages.ts

```diff
@@ -8464,7 +8464,7 @@ export default defineMessages({
     TR_EVM_EXPLANATION_SEND_DESCRIPTION: {
         id: 'TR_EVM_EXPLANATION_SEND_DESCRIPTION',
         defaultMessage:
-            "Send only through {network}. The address must be on the {network} chain or the receiver won't receive their crypto.",
+            'Send only through {network}. The address must be on the {network} chain for the crypto to be received.',
```

Comment:
shouldn't we change it in crowdin?

Reply:
Not sure what is the exact process. Changing it in crowding is also possible (might is the correct way). Let me know and I will add fix up if this issue is a no-go.

Reply:
Please change it in Crowdin, we will then proceed with translations from agency. Thanks!

Reply:
I do not have an access to Crowdin. Please change it, would be much quicker than asking for access. Still, let me know if I should revert the translation in the code or resolve the comment. Thanks.

Reply:
Yes, changing an existing key must be accompanied by a corresponding change in Crowdin, otherwise this will be overwritten during the next Crowdin sync. I don't know if @LukasRada has the permissions to do so so I did it.

https://crowdin.com/editor/trezor-suite/103/en-en?view=comfortable&filter=basic&value=0#q=TR_EVM_EXPLANATION_SEND_DESCRIPTION

Reply:
You can keep the change in code.